src/Uri.php

```diff
@@ -684,13 +684,13 @@ private function filterPath($path): string
             throw new \InvalidArgumentException('Path must be a string');
         }
 
-        /** @var string $thing */
         $thing = preg_replace_callback(
```

Comment:
etc, in all places

Reply:
no, it means adding the var over the return statement itself. you dont need to name the variable.

Reply:
I see. Please take a review again.